routes/family.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
+var router = express.Router();
+var mongoFamily = require('../models/Family').mongoFamily;
+var joiFamily = require('../models/Family').joiFamily;
+
+try {
```

Comment:
you don't put a try/catch around the router like this, you can go to the authentication route and controllers to see how i did it